utils/logger.js

```diff
@@ -29,19 +31,20 @@ const logger = new winston.createLogger({ // eslint-disable-line new-cap
    * Modifications to be made through environment variables defined in config files
    */
   transports: [
-    ...(config.get('enableFileLogs') ? [new winston.transports.File(options.file)] : []),
-    ...(config.get('enableConsoleLogs') ? [new winston.transports.Console(options.console)] : [])
+    ...(config.get('enableFileLogs') ? [new _transports.File(options.file)] : []),
+    ...(config.get('enableConsoleLogs') ? [new _transports.Console(options.console)] : [])
   ],
 
   exitOnError: false // do not exit on handled exceptions
 })
 
 // create a stream object with a 'write' function that will be used by `morgan`
 logger.stream = {
+  // @ts-ignore
```

Comment:
Is this because you were getting errors in the initial setup ??

Reply:
Yes and no, I used `@ts-ignore` when I was checking the setup and converted this file to ts. But as this file is now a js file we can remove it. But I kept it so when we change it to TS we don't get a lot of errors and later fix the type.